machine-learning/_posts/2018-04-26-cost-sensitive-OC.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: review
+title:  "Adaptive Cost-sensitive Online Classification"
+tags:   machine-learning, classification
+author: "Charles Authier"
+pdf: "https://arxiv.org/pdf/1804.02246"
+cite:
+    authors: "Peilin Zhao, Yifan Zhang, Min Wu, Steven C. H. Hoi, Mingkui Tan, Junzhou Huang"
+    title:   "Adaptive Cost-sensitive Online Classification"
+    venue:   "https://arxiv.org/pdf/1804.02246"
+---
+
+## Intro
+They proposed three algorithms and validate their effectiveness and properties for solving cost-sensitive online classification problems.
+They approximate the second covariance matrix Σ by a small number of carefully selective directions.
```

Comment:
"second covariance matrix Σ" ? You have to give more context. So there's a *first* covariance matrix?